kikit/present.py

```diff
@@ -109,9 +113,9 @@ def _renderBoards(self, outputDirectory):
             boardDesc["gerbers"] = os.path.join(dirPrefix, boardName + "-gerbers.zip")
             boardDesc["file"] = os.path.join(dirPrefix, boardName + ".kicad_pcb")
 
-            subprocess.run(["pcbdraw", "--vcuts", "--silent", boardDesc["source"],
+            subprocess.run([pcbdraw, "--vcuts", "--silent", boardDesc["source"],
```

Comment:
Could you also check if the process ended with 0 return code? When Pcbdraw fails for some reason the images will be silently missing, which could be inconvenient in CI. When we report an error, no new page will be deployed and user will be notified via e-mail.

Reply:
@yaqwsx switched `run` calls to `check_call`. If process fails it will raise `subprocess.CalledProcessError`.

Reply:
Fine for me.